src/EFCore.PG/Storage/Internal/Mapping/NpgsqlArrayTypeMapping.cs

```diff
@@ -113,13 +123,16 @@ static ValueComparer CreateComparer(RelationalTypeMapping elementMapping, Type a
             return (ValueComparer)Activator.CreateInstance(typeof(SingleDimComparerWithEquals<>).MakeGenericType(elementType));
         }
 
+        /// <inheritdoc />
```

Comment:
Likewise, seems quite unnecessary for this internal class. Same below.

Reply:
I was referring specifically to the the sense of documenting an internal class such as the various value comparers in NpgsqlArrayTypeMapping.